index.bs

```diff
@@ -1524,16 +1528,35 @@ When this method is invoked, the [=authenticator=] must perform the following pr
     <figcaption>Generating an [=assertion signature=].</figcaption>
 </figure>
 
-On successful completion, the authenticator returns to the user agent:
+On successful completion, the [=authenticator=] returns to the user agent:
 - The identifier of the credential (credential ID) used to generate the [=assertion signature=].
 - The [=authenticator data=] used to generate the [=assertion signature=].
 - The [=assertion signature=].
 
-If the authenticator cannot find any credential corresponding to the specified [=[RP]=] that matches the specified criteria, it
+If the [=authenticator=] cannot find any credential corresponding to the specified [=[RP]=] that matches the specified criteria, it
 terminates the operation and returns an error.
 
 If the user refuses consent, the authenticator returns an appropriate error status to the client.
+    </dd>
+    <dt>"public-key-uaf"</dt>
+    <dd>
+- Follow the procedure as specified in [[FIDO-UAF-Authenticator-Commands]] section 6.3. Using
+    * The caller's RP ID as "AppID"
+    * The [=hash of the serialized client data=] as "Final Challenge Hash".
+    * If present: The content of extension txAuthSimple (i.e. the string) or txAuthGeneric (i.e. the content bytes) as "Transaction Content".
+    * If a list of credentials acceptable to the [=[RP]=] was provided: Provide each {{PublicKeyCredential/rawId}} as "KeyHandle".  If an empty list was provided do not provide any "KeyHandle".
+    * Provide a "KHAccessToken" as described in section 6.1. in [[FIDO-UAF-ASM]].
+    * Provide no "UserVerificationToken".
+- On successful completion, the [=authenticator=] returns to the user agent:
+    * The identifier of the credential used to generate the signature.
+    * binary authenticator generated data according to the structure as specified in [[FIDO-UAF-Authenticator-Commands]] section 5.2.2.
+    * The assertion signature.
+
+If the [=authenticator=] cannot find any credential corresponding to the specified [RP] that matches the specified criteria, it
+terminates the operation and returns an error.
 
+If the user refuses consent, the [=authenticator=] returns an appropriate error status to the client.
+    </dd>
 
```

Comment:
You're missing a `</dl>` here, discovered by looking at the indentation of https://s3.amazonaws.com/pr-preview/w3c/webauthn/credential-type-uaf.html#op-cancel.